src/Resources/public/administration/js/coin-gate-payment.js

```diff
@@ -0,0 +1 @@
tablished successfully."}):e.pluginConfig["CoinGatePayment.config.apiToken"]&&e.createNotificationError({title:"Live API Connection",message:"Connection failed."}),t.sandbox?e.createNotificationSuccess({title:"Sandbox API Connection",message:"Connection established successfully."}):e.pluginConfig["CoinGatePayment.config.apiTokenForSandbox"]&&e.createNotificationError({title:"Sandbox API Connection",message:"Connection failed."}),e.isLoading=!1}))}}})},UCZA:function(e,t){function n(e){return(n="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e})(e)}function o(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}function r(e,t){for(var n=0;n<t.length;n++){var o=t[n];o.enumerable=o.enumerable||!1,o.configurable=!0,"value"in o&&(o.writable=!0),Object.defineProperty(e,o.key,o)}}function i(e,t){return(i=Object.setPrototypeOf||function(e,t){return e.__proto__=t,e})(e,t)}function c(e){var t=function(){if("undefined"==typeof Reflect||!Reflect.construct)return!1;if(Reflect.construct.sham)return!1;if("function"==typeof Proxy)return!0;try{return Boolean.prototype.valueOf.call(Reflect.construct(Boolean,[],(function(){}))),!0}catch(e){return!1}}();return function(){var n,o=a(e);if(t){var r=a(this).constructor;n=Reflect.construct(o,arguments,r)}else n=o.apply(this,arguments);return u(this,n)}}function u(e,t){if(t&&("object"===n(t)||"function"==typeof t))return t;if(void 0!==t)throw new TypeError("Derived constructors may only return object or undefined");return function(e){if(void 0===e)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return e}(e)}function a(e){return(a=Object.setPrototypeOf?Object.getPrototypeOf:function(e){return e.__proto__||Object.getPrototypeOf(e)})(e)}var f=Shopware.Classes.ApiService,s=Shopware.Application,l=function(e){!function(e,t){if("function"!=typeof t&&null!==t)throw new TypeError("Super expression must either be null or a function");e.prototype=Object.create(t&&t.prototype,{constructor:{value:e,writable:!0,configurable:!0}}),Object.defineProperty(e,"prototype",{writable:!1}),t&&i(e,t)}(s,e);var t,n,u,a=c(s);function s(e,t){var n=arguments.length>2&&void 0!==arguments[2]?arguments[2]:"coingate-payment";return o(this,s),a.call(this,e,t,n)}return t=s,(n=[{key:"check",value:function(e){var t="_action/".concat(this.getApiBasePath(),"/verify");return this.httpClient.post(t,e,{headers:this.getBasicHeaders()}).then((function(e){return f.handleResponse(e)}))}}])&&r(t.prototype,n),u&&r(t,u),Object.defineProperty(t,"prototype",{writable:!1}),s}(f);s.addServiceProvider("verifyApiTokenService",(function(e){var t=s.getContainer("init");return new l(t.httpClient,e.loginService)}))},wvpX:function(e,t){e.exports='<div>\n    <sw-button-process :isLoading="isLoading" @click="check">Test API connection</sw-button-process>\n</div>\n'}});
```

Comment:
User would need to manually compile these otherwise. We are including vendor/coingate-php sdk so it makes sense to include a pre-compiled version of custom JS as well. This is needed mainly for "API Test" button to appear and be useable in administration panel right when the plugin is installed.

Reply:
Yes, we include both. Also, documentation recommends to ship the compiled file with a plugin. You compile by running ``psh.phar build`` command within shopware.

![image](https://user-images.githubusercontent.com/1407067/186157876-cce82e70-6e5e-4b93-a927-9ba969d2d4da.png)